db/1.10/modules/ROOT/pages/database-connector-troubleshooting.adoc

```diff
@@ -78,6 +79,60 @@ The following example shows how to use pooling profiles with the *Stored procedu
 	</flow>
 ----
 
+== Understand Select and Stored Procedure Operations Warning Exceptions
+
+When the Database Connector *Select* and *Stored procedure* operations execute correctly but results are not present, Mule runtime engine (Mule) returns a `WARN` exception message instead of throwing the exception.
```

Comment:
```suggestion
When the Database Connector *Select* and *Stored procedure* operations execute correctly but do not return results, Mule runtime engine (Mule) returns a `WARN` exception message instead of throwing the exception.
```